### PR TITLE
CircleCI initial config

### PR DESCRIPTION
**Pull request by ruiposse**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+npm-debug.log
```
Comment (davidpicarra) [inline .gitignore:3]: Should we add npm-debug.log to the gitignore or just not have npm-debug logs at all? 😃

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+machine:
```
Comment (davidpicarra) [inline circle.yml:1]: What exactly does this file do?

File: build/ci.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+set -e  # Exit immediately if a command exits with a non-zero status.
```
Comment (davidpicarra) [inline build/ci.sh:1]: What exactly does this file do? Is it part of a template?

File: package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (davidpicarra) [inline package.json:7]: Should we remove test script or leave it and complete with unit testing in the future?

File: build/ci.sh (hunk 3 above)
Comment (ruiposse) [reply build/ci.sh:1]: This is the file that is called by CircleCI to run commands on each pull-request.

File: package.json (hunk 4 above)
Comment (ruiposse) [reply package.json:7]: For now we should remove it. We will be introducing unit tests very soon as then we can update `package.json` and `ci.sh` (for CircleCI to run our tests as well).

File: circle.yml (hunk 2 above)
Comment (ruiposse) [reply circle.yml:1]: This is the config file for CircleCI to know what to do on each pull-request.

**Review by davidpicarra — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (ruiposse) [reply .gitignore:3]: We could keep deleting it, but this way we prevent the file from being committed by mistake.

**Pull request merged**